source/renderer/app/components/wallet/transactions/Transaction.js

```diff
@@ -300,6 +301,7 @@ export default class Transaction extends Component<Props> {
                   )}
                 >
                   {address}
+                  <SVGInline svg={externalLinkIcon} />
```

Comment:
The icon is not underlined, when hovering

Reply:
@daniloprates it is fixed, nice catch ;)